### ISSUE TITLE
Accessibility: Please change light blue font color to a higher contrast

### ISSUE THREAD

**Issue by normanbrobinson** [ionic-team/capacitor#444]
Contrast Ratio 1.19:1 on your light blue on white background means people with low vision or color blindness are missing out. Please change that to a dark color.

https://webaim.org/resources/contrastchecker/
#a11y

**Comment by mlynch** [comment]
What page are you referring to?

**Comment by jcesarmobile** [comment]
closing since there was no response from the reporter
